catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/LookListFragment.java

```diff
@@ -187,6 +187,7 @@ public void onItemClick(LookData item) {
 		}
 
 		item.invalidateThumbnailBitmap();
+		item.clearCollisionInformation();
```

Comment:
at which point is the collisionInformation then calculated again in the following cases?:
- no changes made / changes discarded
- changes saved